Language/Structure/Control Structure/ternary.adoc

```diff
@@ -0,0 +1,71 @@
+---
+title: ternary
+categories: [ "Structure" ]
+subCategories: [ "Control Structure" ]
+---
+
+
+
+
+
+= ?: Ternary Operator
+
+
+// OVERVIEW SECTION STARTS
+[#overview]
+--
+
+[float]
+=== Description
+It takes three arguments rather than the typical one or two that most operators use. The first argument is a comparison argument, the second is the result upon a true comparison, and the third is the result upon a false comparison. It is an alternative to shorten an if-else block. It can help increase the readability and reduce the number of lines in your code.
+[%hardbreaks]
+
+
+[float]
+=== Syntax
+`expression_1 ? expression_2 : expression_3; // if expression_1 evaluates to true then expression_2 is evaluated else expression_3`
+
+
+[float]
+=== Parameters
+`expression`. An expression is any legal combination of symbols that represents a value.
```

Comment:
```suggestion
`expression`: An expression is any legal combination of symbols that represents a value.
```

Use standard format for parameter documentation:

https://github.com/arduino/reference-en/blob/ac984909057157f045696971da021fa382a1da11/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc?plain=1#L43